src/backend/distributed/commands/trigger.c

```diff
@@ -48,11 +48,13 @@ static bool IsCreateCitusTruncateTriggerStmt(CreateTrigStmt *createTriggerStmt);
 static String * GetAlterTriggerDependsTriggerNameValue(AlterObjectDependsStmt *
 													   alterTriggerDependsStmt);
 static void ErrorIfUnsupportedDropTriggerCommand(DropStmt *dropTriggerStmt);
+static void ErrorOutForTriggerDependsOnExtension(char *triggerName);
```

Comment:
name convention seems to be `ErrorIf....` so something like 
```suggestion
static void ErrorIfTriggerDependsOnExtension(char *triggerName);
```

Reply:
Yeah but this function doesn't have any conditionals in it and it's there to make sure that we throw the same error in different code-paths. So for me it makes more sense to keep this function as is.

Reply:
(will discuss this in https://github.com/citusdata/citus/pull/6399#discussion_r999612019)